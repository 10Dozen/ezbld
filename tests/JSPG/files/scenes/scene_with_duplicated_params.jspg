# SceenWithDuplicatedParams

*goto: "Scene2"
*goto: return Scene3

Some line.

Some line again.

*goto: "Scene1"
