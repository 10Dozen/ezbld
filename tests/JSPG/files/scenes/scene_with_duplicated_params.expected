[0]
cls = JSPGScene
name = SceenWithDuplicatedParams
goto = "Scene1"
desc =
    Some line.

    Some line again.
